spec/helpers/api_helper.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.shared_context "with authenticated requests" do
+  let(:authenticated_user) { user || create(:user) }
+  let(:auth_header) do
```

Comment:
Gonna need to change this because we changes authentication methods. Check out `app/controllers/application_controller.rb`